### PR TITLE
FFmpeg: add rubberband and zimg options

### PR DESCRIPTION
**Pull request by AirPort**
Both are external libraries supported since FFmpeg 3.0.

### CODE REVIEW

File: Library/Formula/ffmpeg.rb [hunk 1]
```diff
@@ -33,6 +33,8 @@ class Ffmpeg < Formula
   option "with-zeromq", "Enable using libzeromq to receive commands sent through a libzeromq client"
   option "with-snappy", "Enable Snappy library"
   option "with-dcadec", "Enable dcadec library"
+  option "with-rubberband", "Enable rubberband library"
+  option "with-zimg", "Enable z.lib library"
```
Comment (mikemcquaid) [inline Library/Formula/ffmpeg.rb]: It's just `zlib`, right? Also, is this library called `zimg` or something?

Comment (AirPort) [inline Library/Formula/ffmpeg.rb]: According to thir repo (https://github.com/sekrit-twc/zimg) is z.lib/zimg

Comment (zmwangx) [inline Library/Formula/ffmpeg.rb]: `zlib` is a well established compression library. Let's go with `zimg` to avoid the confusion.

Comment (zmwangx) [inline Library/Formula/ffmpeg.rb]: Actually I retract my last comment, because https://ffmpeg.org/ffmpeg-all.html calls it "the z.lib library".

Comment (mikemcquaid) [inline Library/Formula/ffmpeg.rb]: I think `z.lib zimg` would be ideal.

Comment (AirPort) [inline Library/Formula/ffmpeg.rb]: Do you mean we should put both names in the description?

Comment (mikemcquaid) [inline Library/Formula/ffmpeg.rb]: Yeh, ideally `"Enable z.lib zimg library"` would be least ambigious

Comment (AirPort) [inline Library/Formula/ffmpeg.rb]: Done.

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**